src/Stryker.Core/Stryker.Core/ProjectComponents/ExcludableString.cs

```diff
@@ -0,0 +1,17 @@
+namespace Stryker.Core.ProjectComponents
+{
+    public class ExcludableString
```

Comment:
What does this class do? And is this cSharp specific?

Reply:
Well, this is now basically just a non-language-specific construct to represent the idea that we can include or exclude files in some code flows. Distinguish `Person.cs` vs `!Person.cs` or `Person.fs` vs `Person.fs`. E.g. `DiffIgnoreChangesInput` don't need info about spans as opposed to `MutateInput`.